developer_guides/debugability/logger/index.rst

```diff
@@ -124,3 +125,70 @@ Examples:
   .. code-block:: bash
 
      sof-logger -l ldc_file -d
+
+
+Trace filtering
+***************
+
+It's possible to change current logs level for any instance of component in
+runtime. To do so, use `-F` option with following argument format:
+
+   <log_level>="<component1>[, <component2>]"
+
+Where *<log_level>* is one of:
+
+- ``c`` / ``critical``
+- ``e`` / ``error``
+- ``w`` / ``warning``
+- ``i`` / ``info``
+- ``d`` / ``debug``
+- ``v`` / ``verbose``
+
+After **=** character, there is a list of component, separated with comma.
+Each components starts with *name* followed by optional part with *instance*
+description.
```

Comment:
How do we list the components again ? We should probably include that here.

Reply:
there such a line in the following paragraph:
 >Try use ``-d`` flag in logger to list component names from `ldc` file content.
Isn't that enough?

Reply:
ok, can you do that and paste the output here too. This way users will know what to expect when reading the later sections. You dont need to paste all the items, just enough to show the structure so everyone has an idea of how to map to the cmd line.